src/app/contribute/guidelines/components/index.html

```diff
@@ -0,0 +1,201 @@
+      </li>
+    </ul>
+    <li>
+      <b>If you make an API change that affects inputs and outputs for a component, we recommend that you provide your potential (not working) documentation and demo to the SKY UX team for review.</b> This lets us head off API issues before implementation. Contributors should keep in mind that the API can change in the course of development.
+    </li>
+  </ul>
+
+  <stache-page-anchor>
+    Localization
+  </stache-page-anchor>
+
+  <p>
+    To declare localization strings, specify a string name and provide the string and a description in the <stache-code>resources_en_US.json</stache-code> file in <stache-code>src/assets/locales</stache-code>. You can reference the localization strings with the <stache-code>skyLibResources</stache-code> pipe or the <stache-code>SkyLibResourcesService</stache-code> service in <stache-code>@skyux/i18n</stache-code>.
+    </p>
+
+  <stache-page-anchor>
+    Get the code
+  </stache-page-anchor>
+
+  <ol>
+    <li>
+      <a href="https://help.github.com/articles/fork-a-repo/">Fork the repo</a> into your own GitHub repo.
+    </li>
+    <li>
+      Create a branch off of master named after the feature you will contribute. The branch name should be kebab-case (my-new-feature).
+    </li>
+    <li>
+      Clone your repo locally and run <stache-code>nvm use</stache-code> to ensure you have the preferred version of Node.js installed. Next, run <stache-code>npm install</stache-code> to install all required dependencies. For more information on prerequisites, see <a stacheRouterLink="/learn/get-started/prereqs">SKY UX prerequisites</a>.
+    </li>
+    <li>
+      <a stacheRouterLink="." fragment="write-visual-regression-tests">Run your first round of visual regression tests with <stache-code>skyux e2e</stache-code></a>. This establishes baseline screenshots of the components in the master repo so that as you make changes and run your visual regression tests, your changes are validated against the state of the repo before the changes.
+    </li>
+  </ol>
+
+  <stache-page-anchor>
+    Write code
+  </stache-page-anchor>
+
+  <ol>
+    <li>
+      Launch a command prompt, <stache-code>cd</stache-code> to the folder where you cloned your branch, and run <stache-code>skyux watch --coverage library</stache-code>.
+    </li>
+    <li>
+      Launch a second command prompt, <stache-code>cd</stache-code> to the folder where you cloned your branch, and run <stache-code>skyux serve</stache-code>. This launches the SKY UX component demos page where you write the demo components to test your code.
```

Comment:
How about something like ...
> Launch a second command prompt, <stache-code>cd</stache-code> to the folder where you cloned your branch, and run <stache-code>skyux serve</stache-code>. This launches the visual testing pages where you can see updates based on changes in your code. Depending on the nature of your changes, you may need to make updates to the visual tests. Keep in mind, any changes here will impact the visual e2e tests, and you may need to reset those baselines after making your changes to get the build to pass.

(maybe add a link to the "Write visual regression tests" section for more context) ?